test/testapps/LiveReloadTestApp/Home.cshtml

```diff
@@ -1 +1 @@
-<h1>Hello, world!</h1>
+<h1>Goodbye, world!</h1>
```

Comment:
This is why the tests failed - the live reloading test should have reverted it back to `Hello` after the test completed (there's logic inside a `finally` block to do that). Not sure why the live reloading test failed to clean up after itself, but if you can repro that please let me know!

Anyway you'll just need to revert this change and hopefully then the build will succeed.